staging/src/k8s.io/apiserver/pkg/authentication/request/x509/x509.go

```diff
@@ -176,6 +177,9 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (*authenticator.R
 	*/
 
 	remaining := req.TLS.PeerCertificates[0].NotAfter.Sub(time.Now())
+	if remaining.Seconds() < 86400 {
+		klog.Warningf("Certificate of client: %s is about to expire in %f seconds", req.RemoteAddr, remaining.Seconds())
```

Comment:
logging on every request seems very verbose, I agree the remote address is likely to not be useful in locating the client, and the remaining time in seconds is difficult to read

would the audit log make more sense as a spot to annotate this? it would already be strongly correlated with the client since user info is in the audit log